executor/index_merge_reader.go

```diff
@@ -820,6 +820,9 @@ func (w *partialIndexWorker) fetchHandles(
 			return count, err
 		}
 		if len(handles) == 0 {
+			if basicStats != nil {
+				basicStats.Record(time.Since(start), chk.NumRows())
```

Comment:
What does the `chk.NumRows()` mean for partialIndexWorker?

Reply:
This line of code is the same as this line in the master:
https://github.com/pingcap/tidb/blob/b91bdd0876dfa566fee6431d2eb59bb9f0ea21e1/executor/index_merge_reader.go#L840

I just copied it here.